src/mux-analytics.brs

```diff
@@ -1,5 +1,5 @@
 sub init()
-  m.MUX_SDK_VERSION = "1.7.1"
+  m.MUX_SDK_VERSION = "1.8.0-dev.1"
```

Comment:
Did you intend to change this before 'shipping?'

Reply:
Yup, good reminder! Will make sure to change before I ship this